keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -208,6 +208,47 @@ type RequeueState struct {
 }
 ```
 
+Add a new field `evictionStats` to the WorkloadStatus struct to track the number of evictions by type and reason:
```

Comment:
Let's use the `evictionState` to align with other fields like `requeueState` and `admisssionCheckState`.

Reply:
sgtm